frontends/infinite-corridor/src/scss/form.scss

```diff
@@ -129,35 +141,6 @@ label.radiolabel {
   font-size: 16px;
 }
 
-input[type="checkbox"],
-input[type="radio"] {
-  box-sizing: border-box;
-  font-size: 16px;
-}
-
-.mdc-checkbox::before,
-.mdc-checkbox::after {
-  background-color: theme.$navy;
-}
-
-.mdc-checkbox__background {
-  background-color: theme.$navy !important;
-  border-color: theme.$navy !important;
-}
```

Comment:
This is from material-web-components, which we aren't currently using in infinite-corridor.

I removed some other unused code in this file, but not all of it. E.g., I left the styling for `select` elements since it seems reasonably likely that we will use it.